sympy/physics/continuum_mechanics/tests/test_beam.py

```diff
@@ -1,7 +1,7 @@
-from sympy import Symbol, symbols, S, simplify
+from sympy import Symbol, symbols, S, simplify, Interval
```

Comment:
May be I'm missing something. Is this import of `Interval` needed?

Reply:
Oh I see why this is needed now. The `max_shear_force` test below wasn't being run before but now you've renamed so it is and needs this.

Reply:
Exactly! There are some similar imports where there are XFAIL:ed tests that was missing imports. Obviously they failed, but now they fail for the right reasons.